2018-03-28T06:17:44.033Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: error executing chaincode: transaction returned with failure: [composerchannel-b298ead6]Calling chaincode Invoke() returned error response [Error: Object with ID '11' in collection with ID 'Asset:org.gov.fundtracker.MMDA' does not exist]. Sending ERROR message back to peer
    at /home/vagrant/hyperledger/server/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-03-28T06:17:44.034Z ERROR   HLFConnection           :queryChainCode()         {"message":"Error trying to query business network. Error: error executing chaincode: transaction returned with failure: [composerchannel-b298ead6]Calling chaincode Invoke() returned error response [Error: Object with ID '11' in collection with ID 'Asset:org.gov.fundtracker.MMDA' does not exist]. Sending ERROR message back to peer","stack":"Error: Error trying to query business network. Error: error executing chaincode: transaction returned with failure: [composerchannel-b298ead6]Calling chaincode Invoke() returned error response [Error: Object with ID '11' in collection with ID 'Asset:org.gov.fundtracker.MMDA' does not exist]. Sending ERROR message back to peer\n    at channel.queryByChaincode.then.catch (/home/vagrant/hyperledger/server/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:820:34)\n    at <anonymous>"}$
